bbp/nse/bbp-morphology-workflow/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  config,
+  fetchgitPrivate,
+  pythonPackages,
+  neurom,
+  neurodamus,
+  bbpsdk-legacy,
+  muk,
+  bluerepairsdk,
+  morphscale,
+  bluejittersdk,
+}:
+
+pythonPackages.buildPythonPackage rec {
+    pname = "bbp-morphology-workflow";
+    version = "0.0.3.dev1";
+    name = "${pname}-${version}";
+
+    src = fetchgitPrivate {
+        url = config.bbp_git_ssh + "/platform/bbp-morphology-workflow";
+        rev = "4808570b7914dc621db45a2709f62cfcea73d57c";
+        sha256 = "0adll1vz0nb1y9rrmv8w8r8h66pb9nb4id1vidpxs0aa9j19cba5";
+    };
+
+    checkInputs = with pythonPackages; [
+        pylint
+        coverage
+        pycodestyle
+        mock
+        nose
+    ];
+
+    propagatedBuildInputs = with pythonPackages; [
+        neurom
+        neurodamus
+        bbpsdk-legacy
+        muk
+        bluerepairsdk
+        morphscale
+        bluejittersdk
+    ];
+
+    checkPhase = ''
+        nosetests morphology_repair_workflow/tests
```

Comment:
You should keep the pre/post hook:

```
checkPhase = ''
  runHook preCheck
  nosetests morphology_repair_workflow/tests
  runHook postCheck
'';

Reply:
what's that for?

Maybe it would be worthwhile to add tips/information like this to a nix page?

Reply:
Here is the Nix documentation about the [check phase](https://nixos.org/nixpkgs/manual/#ssec-check-phase)
`preCheck` and `postCheck` could be defined elsewhere.